### DISCUSSION TITLE
Chart/ page/ summarization for whole market

### DISCUSSION THREAD

**Discussion by moorgrove** [electricitymaps/electricitymaps-contrib#7537] [Ideas]
It would be nice to have a page or some sort of chart to compare countries/ regions in connected markets. One example is to easily compare the "hourly net exchange" for all countries in Europe to see who over time have net export or net import. Instead of going thru one country at a time, have them visualized in one page.

**Comment by VIKTORVAV99** [comment]
That it would!

The teams have a few different priorities at the moment but we'll keep this in mind for the future!
